lib/pg_party/adapter_decorator.rb

```diff
@@ -419,8 +425,8 @@ def hashed_table_name(table_name, key)
 
     def index_valid?(index_name)
       select_values(
-        "SELECT relname FROM pg_class, pg_index WHERE pg_index.indisvalid = false AND "\
-          "pg_index.indexrelid = pg_class.oid AND relname = #{quote(index_name)}"
+        "SELECT relname FROM pg_class, pg_index WHERE pg_index.indisvalid = false AND " \
+        "pg_index.indexrelid = pg_class.oid AND relname = #{quote(index_name)}"
```

Comment:
This one should probably just use heredoc

Reply:
Done